examples/04_training/06_train_icemix_model.py

```diff
@@ -0,0 +1,277 @@
+"""Example of training Model.
+
+This example is based on Icemix solution proposed in
+https://github.com/DrHB/icecube-2nd-place.git (2nd place solution). There are a
+few things to note about this example:
```

Comment:
Looks like you forgot to finish this docstring

Reply:
Removed. 
For further familiarization with the solution I think is enough with #681 .